envs/yocto/shell.nix

```diff
@@ -37,23 +38,23 @@ let
         # TODO limit export to native pkgs?
         nixconf = pkgs.writeText "nixvars.conf" ''
           # This exports the variables to actual build environments
-          # From BB_ENV_EXTRAWHITE
+          # From BB_ENV_PASSTHROUGH_ADDITIONS
```

Comment:
Could you keep the previous one, as it allows use with older releases?

Reply:
ok